src/FSharp.CodeFormat/FSharp.CodeFormat.fsproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>FSharp.CodeFormat</RootNamespace>
     <AssemblyName>FSharp.CodeFormat</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
@forki Do you think that updating to .NET 4.5 could cause any troubles?

I was worried about this when it came up in the discussion with @matthid, but I don't think it can cause troubles to anyone (the main usage of F# Formatting is through ProjectScaffold and I think that should most likely be fine...)